ai/api_config.py

```diff
@@ -0,0 +1,38 @@
+import os
+from logging import Logger
+
+API_KEY_DICT= {
+    "openai": os.environ.get("OPENAI_API_KEY"),
+    "anthropic": os.environ.get("ANTHROPIC_API_KEY"),
+    #"your-api-name": os.environ.get("YOUR_API_KEY")
+}
+APIS_AND_MODELS = {
+    "OpenAI": ["GPT-4o", "GPT-4 Turbo", "GPT-4", "GPT-3.5 Turbo"],
+    "Anthropic": ["Claude 3.5 Sonnet", "Claude 3 Sonnet", "Claude 3 Haiku", "Claude 3 Opus"],
+    #"Your-Api-Name": ["Your-Api-Models"]
+}
+
+# Utility functions
+def isAPIKeySet(api_name: str) -> bool:
+    try:
+        return API_KEY_DICT.get(api_name.lower())
+    except Exception as e:
+        print(e)
+
+
+
+def getAvailableAPIsAndModels():
```

Comment:
Nit: maybe we should `snake_case` everything

```suggestion
def get_available_apis_with_models():
```